test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -115,5 +115,23 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains("/Views/ErrorPageMiddleware/RuntimeError.cshtml", content);
             Assert.Contains(expectedMessage, content);
         }
+
+        [Fact]
+        public async Task LoaderExceptionsFromReflectionTypeLoadExceptionsAreListed()
+        {
+            // Arrange
+            var expectedMessage = "Failure has occurred while loading a type.";
+            var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
+
+            // Act
+            var response = await Client.GetAsync("http://localhost/LoaderException");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
+            Assert.Equal(expectedMediaType, response.Content.Headers.ContentType);
+            var content = await response.Content.ReadAsStringAsync();
+            Assert.Contains("LoaderExceptions:", content);
```

Comment:
Might be better to add a message to the loader exception you added and verify that. Also, did we not change the text for this string to say "Loader Exceptions" (with the space?)